netbox_netprod_importer/push.py

```diff
@@ -323,7 +323,7 @@ def _handle_device(self, hostname, importer, discovered, overwrite):
                     logger.debug("True with interco %s:", interco)
                 except Exception as e:
                     result["errors"] += 1
-                    logger.debug("Error with interco %s: %s", interco, e)
+                    logger.warning("Switch %s Error with interco %s: %s",hostname ,interco, e)
```

Comment:
It's a good idea, but I avoided it before because it really floods the user with a lot of errors. What is your opinion about it?

Reply:
Indeed, a lot of mistakes scares. By default, I made the ERROR level, maybe I should set it to CRITICAL. But this is necessary for tracking unidentified devices, perhaps not found devices need to be transferred to INFO. But informing is a must, albeit with an additional option.